swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -90,24 +90,27 @@ public class Adapter {
   private var token: String
 
   // Docs on filter strings: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html
+  private let logFilter: String
   #if DEBUG
-    private let logFilterString =
+    private let defaultLogFilter =
       "connlib_client_apple=debug,firezone_tunnel=trace,connlib_shared=debug,connlib_client_shared=debug,warn"
   #else
-    private let logFilterString =
+    private let defaultLogFilter =
       "connlib_client_apple=info,firezone_tunnel=info,connlib_shared=info,connlib_client_shared=info,warn"
   #endif
```

Comment:
These are duplicated from AdvancedSettings -- can we keep them defined in one place to make them easier to find and maintain?

Reply:
This redundancy is a safeguard -- what if for some reason the main app hasn't set the log filter. Also, this can happen when the app is using a tunnel that was created by the previous version of the app (which doesn't set the logFilter field in the tunnel configuration) (this will not be an issue in production though).

Reply:
On second thoughts, I do think I can handle this differently and avoid the duplication.

Reply:
> what if for some reason the main app hasn't set the log filter

This reminds me -- we should validate the string is non-blank / empty.

Reply:
> we should validate the logFilter string is non-blank / empty

We do validate it. The 'Apply' / 'Save' is disabled if the logFilter is empty.

Reply:
The logFilter string duplication is fixed in commit [c250e02](https://github.com/firezone/firezone/pull/2540/commits/c250e025ef889d78fce9576000262947dbd66341).